ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml

```diff
@@ -20,14 +20,10 @@
     default: "@aws-ipi"
     documentation: "Executed cucumber tags"
   - name: E2E_SKIP_TAGS
-    default: "not @console
```

Comment:
Can we trust console cases now?

Reply:
@jhou1  According to the test results in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/27625/rehearse-27625-periodic-ci-openshift-verification-tests-master-stable-4.9-e2e-aws-ipi/1511867092322750464, I can see many console test cases passed.

Reply:
Thanks!